CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Packaging
 
 * This library now requires a minimum deployment target of iOS 12.0 or above, macOS 10.14.0 or above, tvOS 12.0 or above, or watchOS 5.0 or above. Older operating system versions are no longer supported. ([#736](https://github.com/mapbox/mapbox-directions-swift/pull/736))
+* MapboxDirections now requires [Turf v2.6.0-beta.1](https://github.com/mapbox/turf-swift/releases/tag/v2.6.0-beta.1). ([#757](https://github.com/mapbox/mapbox-directions-swift/pull/757))
```

Comment:
After upgrade to Turf v2.6.0-beta.1 that contains such [changes](https://github.com/mapbox/turf-swift/compare/v2.5.0...v2.6.0-beta.1). Carthage build job that uses Xcode 14 started failing (e.g. https://app.circleci.com/pipelines/github/mapbox/mapbox-directions-swift/1805/workflows/6386f69a-33db-479b-924d-29858d2d09e7/jobs/8067). 

Even though error is not really descriptive I do not think that issue is caused by increasing minimum deployment targets. @OdNairy, would you be able to comment on this one? I think that issue is most likely caused by https://github.com/mapbox/turf-swift/pull/197

Reply:
There are outdated schemes like `Turf TV`. Can you remove them, please?
<img width="214" alt="image" src="https://user-images.githubusercontent.com/735178/197486025-b898581c-cca1-4de2-a5c9-53d926283331.png">